bundle/src/main/java/com/cognifide/cq/cqsm/core/antlr/type/ApmString.java

```diff
@@ -37,4 +37,9 @@ public String getString() {
   public String getValue() {
     return value;
   }
+
+  @Override
+  public String toString() {
+    return String.format("'%s'", getValue());

```

Comment:
You really need to do that? I mean 'getValue()' returns the same as overriden 'toString()'

Reply:
Not the same, here you will get value in apostrophes. I did that to get this effect:
```Defined variable string = 'string'
Defined variable number = 10
Defined variable boolean = false```

Reply:
oh, I see, I didn't notice apostrophes. My bad